docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md

```diff
@@ -3,60 +3,51 @@ title: Using Visual Studio Tools for Docker (Visual Studio on Windows)
 
-If you're using Visual Studio 2015, you must have Update 3 or a later version plus the Visual Studio Tools for Docker.
-
-**More info:** For instructions on installing Visual Studio, go to [https://visualstudio.microsoft.com/\
-products/vs-2015-product-editions](https://visualstudio.microsoft.com/products/vs-2015-product-editions).
+**More info:** For instructions on installing Visual Studio, go to [https://visualstudio.microsoft.com/downloads/](https://visualstudio.microsoft.com/downloads/).
 
 To see more about installing Visual Studio Tools for Docker, go to <http://aka.ms/vstoolsfordocker> and <https://docs.microsoft.com/aspnet/core/host-and-deploy/docker/visual-studio-tools-for-docker>.
 
-If you're using Visual Studio 2017, Docker support is already included.
-
-## Using Docker Tools in Visual Studio 2015
-
-The Visual Studio Tools for Docker provides a consistent way to develop and validate locally your Docker containers for Linux in a Linux Docker host or VM, or your Windows Containers directly on Windows.
-
-If you're using a single container, the first thing you need to begin is to turn on Docker support into your .NET Core project. To do this, right-click your project file, as shown in Figure 4-25.
-
-![https://i1.visualstudiogallery.msdn.s-msft.com/0f5b2caa-ea00-41c8-b8a2-058c7da0b3e4/image/file/205468/1/add-docker-support.png](./media/image31.png)
-
-Figure 4-25: Turning on Docker support for your Visual Studio project
+Docker support is included in Visual Studio 2017.
 
 ## Using Docker Tools in Visual Studio 2017
 
-When you add Docker support to a service project in your solution (see Figure 4-26), Visual Studio is not just adding a DockerFile file to your project, it also is adding a service section in your solution's docker-compose.yml files (or creating the files if they didn't exist). It's an easy way to begin composing your multicontainer solution; you then can open the docker-compose.yml files and update them with additional features.
+When you add Docker support to a service project in your solution (see Figure 4-26), Visual Studio adds a *Dockerfile* to your project without adding support for multiple containers. This configuration is ideal for single-project solutions with a single container.
 
-![](./media/image32.png)
+![Turning on Docker Solution support in a Visual Studio 2017 project](./media/image32.png)
 
 Figure 4-26: Turning on Docker Solution support in a Visual Studio 2017 project
 
-This action not only adds the DockerFile to your project, it also adds the required configuration lines of code to a global docker-compose.yml set at the solution level.
+With Visual Studio version 15.8 and later, if you have multiple projects in a solution that each have an associated container, you should consider adding Docker Compose functionality.  To add this functionality, right-click on the solution or project node, and choose **Add > Container Orchestration Support**.
```

Comment:
- you should consider --> consider
- Don't bold the > character, since it's not a UI element.
- There are 2 spaces after the period. Change this to 1 space.
- Indicate the right-click is needed in Solution Explorer. Bold Solution Explorer, since it's a UI element.
- Service Fabric is an alternative to Docker Compose for container orchestration and is supported by the tooling. Should we mention that option too?